### ISSUE TITLE
No member false positive for Mido

### ISSUE THREAD

**Issue by Void2258** [PyCQA/pylint#1945]
### Steps to reproduce
```
import mido
Names = mido.get_input_names()
with mido.open_input(Names[1]) as padin:
	for msg in padin: #get continuous signals (I am aware this has no programmatic exit)
		print(msg)
```
#run pylint

### Current behavior
E1101:Module 'mido' has no 'get_input_names' member
E1101:Module 'mido' has no 'open_input' member
probably happens for everything in the library, but have only run these two.

### Expected behavior
No error
Note that program will run fine if midi input is connected and valid.

**Comment by PCManticore** [comment]
This error happens because `mido` adds its member in a dynamic way by modifying the `globals()`: https://github.com/olemb/mido/blob/master/mido/__init__.py#L113. It's a behaviour that will not support, since this is not amenable to static analysis. The alternatives here is to:

1. disable analysing the `mido` module by using `ignored-modules=mido`
2. generate additional members for `mido` by using `generated-members=get_input_names,open_input`

**Comment by Void2258** [comment]
How do you set pylint to ignore the module? I can't find where `ignored-modules=mido` is supposed to go.
